pkg/devfile/parser/parse_test.go

```diff
@@ -41,7 +45,7 @@ func Test_parseParentAndPluginFromURI(t *testing.T) {
 		Data: &v2.DevfileV2{
 			Devfile: v1.Devfile{
 				DevfileHeader: devfilepkg.DevfileHeader{
-					SchemaVersion: schemaV200,
+					SchemaVersion: schemaV210,
```

Comment:
should be `v2.2.0`, the bool pointer change is not in v2.1.0. 
we should also update all other tests to test with latest schema. probably create a file for const values will be used in tests, including latest_schema, isTrue and isFalse, and etc. 
 it doesn't have to be within this PR, we can create another issue to track that.

Reply:
The bool pointer updates do not affect the schema structure, just the API.   I needed to bump up the schema versions so we could test the `ephemeral` property which did not exist in v2.0.0.  We can consider refactoring the bool variables and possibly other utils to a common file for all tests (unit, functional) to reference.